testutil/testutil.go

```diff
@@ -181,6 +183,20 @@ func GetTestNomadCreds(t *testing.T) (string, string) {
 	return v[0], v[1]
 }
 
+func CheckTestVaultVersion(t *testing.T) bool {
+	v := SkipTestEnvUnset(t, "TF_VAULT_VERSION")
+
+	cutoffVersion, _ := goversion.NewVersion("1.11")
```

Comment:
Wondering if it makes sense to add a constant for the latest Vault version as opposed to hard-coding. Might make it easier to update after new version are released?

Reply:
Ah yeah that makes sense. In that case, do you think the `CheckTestVaultVersion` function should accept a second parameter, the Vault version to use as `cutoffVersion`? 
```
func CheckTestVaultVersion(t *testing.T, string cutoff) bool {
   ⋮
   cutoffVersion, _ := goversion.NewVersion(cutoff)
```

Reply:
That sounds like a great idea!

Reply:
Updated https://github.com/hashicorp/terraform-provider-vault/pull/1518/commits/f7bde28d4de4f4f657c25f16fef9daea43bb5a44